lib/internal/errors.js

```diff
@@ -178,6 +178,16 @@ E('ERR_TRANSFORM_WITH_LENGTH_0',
 E('ERR_UNKNOWN_SIGNAL', 'Unknown signal: %s');
 E('ERR_UNKNOWN_STDIN_TYPE', 'Unknown stdin file type');
 E('ERR_UNKNOWN_STREAM_TYPE', 'Unknown stream file type');
+E('ERR_SOCKET_CLOSED', 'Socket is closed');
+E('ERR_TLS_DH_PARAM_SIZE', (size) =>
+  `DH parameter size ${size} is less than 2048`);
+E('ERR_TLS_HANDSHAKE_TIMEOUT', 'TLS handshake timeout');
+E('ERR_TLS_SESSION_ATTACK', 'TSL session renegotiation attack detected');
+E('ERR_MULTIPLE_CALLBACK',
```

Comment:
Thanks for the suggestion @refack 
I have redefined the error message accordingly but this lead to change of another test file: `test-stream-transform-callback-twice.js`
Hope that is fine with this PR.
Thank you!